chaincfg/params.go

```diff
@@ -288,6 +289,8 @@ var MainNetParams = Params{
 
 	CosmicInflationActivationTime: 1652616000,
 
+	Upgrade9ForkHeight: 792772, // 000000000000000002B678C471841C3E404EC7AE9CA9C32026FE27EB6E3A1ED1
```

Comment:
not sure whether you use this with `>` or `>=`, just to double-check: 772 was the last block with old rules, 773 is the first block with new rules

Reply:
In `rpcserver`, `mempool` and `process`, all use `>`

e.g.
```go
if height > s.cfg.ChainParams.Upgrade9ForkHeight {
	upgrade9Active = true
}
```